spec/licensee/license_spec.rb

```diff
@@ -421,4 +421,41 @@
       end
     end
   end
+
+  context 'source regex' do
+    Licensee::License.all(hidden: true, psuedo: false).each do |license|
+      context "the #{license.title} license" do
+        let(:source) { URI.parse(license.source) }
+
+        %w[http https].each do |scheme|
+          context "with a #{scheme}:// scheme" do
+            before { source.scheme = scheme }
+
+            ['www.', ''].each do |prefix|
+              context "with '#{prefix}' before the host" do
+                before do
+                  source.host = "#{prefix}#{source.host.sub(/\Awww\./, '')}"
+                end
+
+                ['.html', '.htm', '.txt', ''].each do |suffix|
+                  context "with '#{suffix}' after the path" do
+                    before do
+                      next if license.key == 'wtfpl'
```

Comment:
Is this because the source URL listed for WTFPL doesn't include the license text?

Seems that should be fixed in choosealicense.com, probably http://www.wtfpl.net/txt/copying/ per https://spdx.org/licenses/WTFPL.html